distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogSearchUi.java

```diff
@@ -245,11 +283,93 @@ public void testWorkspaceQueriesWithSpecificSources() {
     List<Map<String, Object>> queries = ImmutableList.of(query);
     Map<String, Object> workspace = ImmutableMap.of(WORKSPACE_QUERIES, queries);
 
-    Response res = expect(asAdmin().body(stringify(workspace)), 201).post(api());
+    Response res = expect(asAdmin().body(stringify(workspace)), 201).post(workspacesApi());
 
     Map body = parse(res);
     String id = (String) body.get("id");
     assertNotNull(id);
     assertThat(body.get(WORKSPACE_QUERIES), is(queries));
   }
+
+  @Test
+  public void testGetSystemTemplates() {
+    Set<String> expectedQueryTemplateTitles =
+        new HashSet<>(ImmutableSet.of("Contact Name", "Imagery Only"));
+
+    Response httpRes1 = expect(asUser("srogers", "password1"), 200).get(queryTemplatesApi());
```

Comment:
where did this username and password come from?

Reply:
This is one of our well known test users. I did not add him. Refer to: 
https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/resources/etc/test-users.properties#L6